crates/phala-mq/src/send_queue.rs

```diff
@@ -105,31 +114,32 @@ mod msg_channel {
                 signer,
             }
         }
+    }
 
-        fn send_data(&self, payload: Vec<u8>, to: impl Into<Path>) {
+    impl<Si: MessageSigner + Clone> MessageChannel<Si> {
+        fn prepare_with_data(
```

Comment:
It's not super clear what preparation is here. So I think it's better to document it a bit. My understanding is to pack all the necessary information, ready to sign, and pass to SideTask api for sending out. Is it correct?

Reply:
Added some comments on the two traits.